packages/grafana-ui/src/components/Space/Space.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { css, cx } from '@emotion/css';
+import { GrafanaTheme2 } from '@grafana/data';
+import { useTheme2 } from '../../themes';
+
+export interface SpaceProps {
+  v?: number;
+  h?: number;
+  layout?: 'block' | 'inline';
+}
+
+export const Space = (props: SpaceProps) => {
```

Comment:
not thrilled witih the name given the single namespace of grafana/ui 
alternatives
* LayoutSpace 
* LayoutSpacing

would probably then add some flex options as well, display: 'flex' | 'block' | 'inline' , etc and justifyContent & alignItems

Reply:
I think the name Space is fine :) It's simple, does the job, and don't think there's confusion about what it does. But also, it's just a name so I don't care if you feel more strongly in favor of something else.

I'm not sure if any flex options are appropriate though - his should not contain any children and should be used only to provide a fixed amount of space. There could be room for like a `<FlexItem grow={1} />` element that could contain children. 

I do think some other layout primitives would be useful, but I want to be careful in not overloading this one component, and making things that can be composed together easily.